activestorage/app/models/active_storage/variant.rb

```diff
@@ -91,6 +91,14 @@ def filename
     ActiveStorage::Filename.new "#{blob.filename.base}.#{variation.format.downcase}"
   end
 
+  def content_type_for_serving
```

Comment:
since `Preview` `VariantWithRecord` and `Variant` are supposed to "have the same interface/contract"

I feel this contract has been  broken in a while ( For instance: `Preview#filename` is not defined as well) .

 However, should i add these 2 new methods on those classes too? @jonathanhefner